### DISCUSSION TITLE
How to set the parameters for the interfacial bonds in composite materials.

### DISCUSSION THREAD

**Discussion by hallojie** [PeriHub/PeriLab.jl#214] [Q&A]
I would like to know if there are any settings here regarding the interfacial bonding properties of materials.

**Comment by CWillberg** [comment]
Hello,

There is an option. You can find an example here

https://github.com/PeriHub/PeriLab.jl/blob/main/examples/Undulations/Smetana.yaml

      dam_1:
        Critical Value: 2.5e-2
        Damage Model: Critical Energy
        Interblock Damage:
          Interblock Critical Value 1_2: 2.5e-4
          Interblock Critical Value 2_3: 2.5e-4
         ....

You can choose the value as you like.

If you need additional documentation. Please add an issue
https://github.com/PeriHub/PeriLab.jl/issues/new/choose

Christian
